drivers/sensor/qdec_mcux/qdec_mcux.c

```diff
@@ -198,6 +202,12 @@ static void init_inputs(const struct device *dev)
 		ENC_GetDefaultConfig(&data->qdec_config);			\
 		data->qdec_config.decoderWorkMode = int_to_work_mode(		\
 			DT_INST_PROP(n, single_phase_mode));			\
+		QDEC_SET_COND(n, data->qdec_config.filterCount, filter_count);	\
+		QDEC_SET_COND(n, data->qdec_config.filterSamplePeriod,		\
+			  filter_sample_period);				\
```

Comment:
> Maybe simpler to use `DT_INST_PROP_OR` instead

True. I used the C_COND_CODE wrapper define to avoid overwriting the default values ENC_GetDefaultConfig sets. Would you like me to change it?

Reply:
Oh, I see. No you don't need to change it